### DISCUSSION TITLE
Permissions 2

### DISCUSSION THREAD

**Discussion by Officepoint** [rejetto/hfs#475] [General]
Hi!
I have this settings from the link below which applies to HOME folder.
https://github.com/rejetto/hfs/wiki/Permissions

Examples
Make folders not accessible by default
If you want your folders to not be accessible unless you give access to a specific account
•	click on Virtual File System > home
o	permissions are inherited, so setting something on the home can affect all the rest
•	**set "Who can download" to "none"
o	prevent access
•	set "Who can list" to "any account"**
o	otherwise you cannot even enter the home page
o	because of any-account, you will automatically get the login dialog. If you don't want that, you can set "anyone" instead.

Case 1 
How can I add mp3 or jpg files from several separate folders into a virtual folder to create a playlist?
I tried to add mp3 files from several folders but only one folder remains set when i use ”load content from disk” .

Case 2 
How do I set permissions for a single mp3 file or picture, from virtual folder, for a single user who does not have an account, when i already have set the permissions below to HOME:
•	**set "Who can download" to "none"
o	prevent access
•	set "Who can list" to "any account"**
I want that user to listen only that single song or view only that single picture without having to create an account for him.

Thank you!

**Comment by rejetto** [comment]
1: if you want to mix files from different folders, you can by adding each one under the destination folder

2: if the single file you want to set is NOT in the VFS (because it's from "load content from disk"), your best option is to add it (click on the folder, then click add and pick the file), so you can easily change permissions now.
Otherwise you must use the "masks" field.
